.github/workflows/docusaurus.yml

```diff
@@ -0,0 +1,14 @@
+name: docusaurus_build
+# This workflow is triggered on a pull request. This is a single event so no brackets are needed.
+on: pull_request
+
+jobs:
+  build:
+    # Configures a build matrix containing multiple operating systems. For each configuration, a copy of the job runs and reports a status.
+    runs-on: ${{ matrix.os }}
+    # To specify the array of operating systems, we must list them under 'strategy'.
+    strategy:
+      matrix:
+        # This job runs on the latest version of Linux.
+        os: [ubuntu-latest]
+    steps:
```

Comment:
I don't see any steps for this action to perform, does this really publish the site that is built to gh-pages?

Reply:
That's why I made this PR as a draft version, I'm now figuring out the use of steps :).

Reply:
Oh, ok. You already asked a review, so that's what I did. ;)

Reply:
Oops, I thought you would only get a notification if I changed the status from draft to 'ready for review'!